README.md

```diff
@@ -93,9 +93,9 @@ DRAKVUF Sandbox may optionally draw a behavioral graph using [ProcDOT](https://w
 1. [Download ProcDOT](https://www.procdot.com/downloadprocdotbinaries.htm) (Linux version).
 2. With your downloaded `procdot*_linux.zip` archive, execute the following commands:
    ```
-   unzip -o procdot*_linux.zip lin64/* -d /tmp/procdot
-   mv /tmp/procdot/lin64 /opt/procdot
-   chmod +x /opt/procdot/procmon2dot
+   # unzip -o procdot*_linux.zip lin64/* -d /tmp/procdot
```

Comment:
`$` ?

Reply:
Although this is technically correct (the best kind of correct) I don't see any value of changing $/# between commands in a block :thinking: